types/commonmark/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for commonmark.js 0.27
+// Type definitions for commonmark.js 0.28
```

Comment:
It looks like this change is valid for 0.27 too. The version number can be left at 0.27.